env_spec_test.py

```diff
@@ -2,17 +2,23 @@
 
 
 def test_invalid_input_non_latin():
-    assert env_spec.render_env_spec_to_html("dαtabase_url") == ""
+    assert env_spec.render_env_spec_to_html(["dαtabase_url"]) == ""
 
 
 def test_invalid_input_start_with_digit():
-    assert env_spec.render_env_spec_to_html("1DATABASE_URL") == ""
+    assert env_spec.render_env_spec_to_html(["1DATABASE_URL, url"]) == ""
+
+
+def test_non_existed_type():
+    assert env_spec.render_env_spec_to_html(["DATABASE_URL, testing"]) == ""
 
 
 def test_valid_input():
-    assert env_spec.render_env_spec_to_html(["DATABASE_URL", "ADMIN_EMAIL"]) == (
+    assert env_spec.render_env_spec_to_html(
+        [["DATABASE_URL", "url"], ["ADMIN_EMAIL", "email"]]
```

Comment:
I would like to see mixed input here. Some lines should declare types, while others no. Example:

```
DATABASE_URL: url
ADMIN_NAME
ADMIN_EMAIL: email
```